.github/workflows/format_code.yml

```diff
@@ -0,0 +1,14 @@
+name: Auto-format code on every PR
+
+on:
+  push:
+  pull_request:
+    branches: [main]
```

Comment:
here the branch should be `[master]` instead of `[main]`

Reply:
Done